k8sutils/pkg/consts/consts.go

```diff
@@ -69,4 +69,9 @@ var (
 const (
 	OdigosCloudApiKeySecretKey = "odigos-cloud-api-key"
 	OdigosOnpremTokenSecretKey = "odigos-onprem-token"
+)
+
+const (
+	OdigosUiServiceName = "ui"
+	OdigosUiServicePort = 3000
```

Comment:
great!

Can you please also update [here](https://github.com/odigos-io/odigos/blob/main/cli/cmd/resources/ui.go)